mod/quiz/accessrule/seb/rule.php

```diff
@@ -303,7 +303,12 @@ public function prevent_access() {
             return $this->get_require_seb_error_message();
         }
 
-        if ($this->accessmanager->should_validate_config_key() && !$this->accessmanager->validate_config_key()) {
+        if ($this->accessmanager->validate_seb_keys() === false) {
```

Comment:
just wondering why we check  === false here. and not doing the same in if ($this->accessmanager->get_session_access())?

Reply:
Changed $this->accessmanager->get_session_access() to be consistent. Prefer adding the equality for readability.